tyk-stack/values.yaml

```diff
@@ -104,6 +105,11 @@ global:
     # DevPortalLicense - Tyk Developer Portal License
     # DevPortalStorageConnectionString - Tyk Developer Portal connection string to the selected database.
     useSecretName: ""
+    # devPortal secret stands for secret including Tyk Dashboard credentials.
```

Comment:
Can we add a description of the use-case of this secret? Although it is visible the content of the secret, the secret's underlying use-case is obvious.

Reply:
Fixed in [f495728](https://github.com/TykTechnologies/tyk-charts/pull/165/commits/f495728457472787620d7d3646c0523017ea19a9)